Cargo.lock

```diff
@@ -2281,7 +2297,7 @@ version = "0.9.53"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "465d16ae7fc0e313318f7de5cecf57b2fbe7511fd213978b457e1c96ff46736f"
 dependencies = [
- "autocfg",
+ "autocfg 0.1.7",
```

Comment:
FWIW, if you also update to `openssl-sys 0.9.54`, then everything will be on `autocfg 1.0.0`.